codecov.yml

```diff
@@ -0,0 +1,11 @@
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+
+coverage:
```

Comment:
Also, could you please add comment here to explain the current coverage criteria? with an example in your design doc will be better.